### PR TITLE
feat(map):  re-center map on active area.

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#682

screen shot 2015-07-23 at 12 27 08 pm
screen shot 2015-07-23 at 12 27 46 pm
screen shot 2015-07-23 at 12 28 02 pm

### CODE REVIEW

**Comment by ceoaliongroo** [comment]
__Update:__
screen shot 2015-07-23 at 11 17 45 am

**Comment by ceoaliongroo** [comment]
@niryariv, the red border is informative.  Do you think is worthy to create a parameter to configure  to hide and show the frame of the `active area of the map`, for testing or development only?

**Comment by niryariv** [comment]
@ceoaliongroo up to you - if you think it's useful for development, go for it

File: client/app/styles/_leaflet-directive.scss [hunk 1]
```diff
@@ -10,3 +10,43 @@
...
+  bottom: 10px;
+}
+
+.border {
+  position: absolute;
+  border: 2px solid Brown;
+  z-index: 2;
+}
+
+.b1 {
```
Comment (amitaibu) [inline client/app/styles/_leaflet-directive.scss]:
```scss
#map {

   .map-border-top
}
```
too general

**Comment by ceoaliongroo** [comment]
ok, for now i moved the code to a [gist](https://gist.github.com/ceoaliongroo/9eb9f63340fc9b1d5210), depend how much we, it's possible to re-add.

**Comment by ceoaliongroo** [comment]
Merged.

**Pull request closed without merging**